imagededup/methods/hashing.py

```diff
@@ -192,6 +193,7 @@ def _find_duplicates_dict(
         max_distance_threshold: int = 10,
         scores: bool = False,
         outfile: Optional[str] = None,
+        search_method: str = 'brute_force_cython' if not os.name == 'nt' else 'bktree',
```

Comment:
In setup.py and tests, sys.platform() has been used instead of os.name. Let's stick to one uniform method to derive the os type.